### DISCUSSION TITLE
Question about the second issue

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#246] [Q&A] [answered]
If we need to add the `inflation allowance` to the inflation asset contract, that is, to build a second issue action, do we need to keep the contract id the same as the old one(the Genesis contract id)?

**Comment by cymqqqq** [comment]
Then, will the `outpoint of the asset owner` and the `outpoint of the inflation allowance` be the same or different?

**Comment by dr-orlovsky** [comment, marked as answer]
Sorry, you completely misunderstand how things work. You do not add inflation allowance to a contract, contracts are immutable.

You need a proper schema supporting that - and this is not anyhow related to the contract id.

Side-note: It is very funny how people blaming RGB and/or claiming that they will do a better RGB at the same time do not know the very basics of RGB and how it works, and come here to ask...

**Reply by cymqqqq** [reply]
We both agree that the contract is immutable, and yes, I need a schema. So what I mean here is that I generated an InflatableAsset schema, and its interface is defined as `RGB20Inflatable`.  Then I need to add the global states and an owned state(assignments) that are the `inflation allowance`, and then finish the contract issuance(so we can get the contract id after the contract issuance).

**Reply by cymqqqq** [reply]
Side-note: there is a little bit of information about the RGB protocol(Or maybe needs to update), I read the `general idea` of this discussion and the source code again and again and again......If I have some questions, so I will come here to ask.

**Reply by dr-orlovsky** [reply]
There is plenty of information about RGB we created over the years: hundreds of hours of developer-educating videos at Plan B Network courses and LNP/BP youtube; https://rgb.tech and https://rgb.info websites, and so on. But of course instead of learning it is easier to spam with useless PRs, issues, and the same questions asked hundreds of times before instead of reading - and also criticizing on Twitter how RGB is made.

Sorry, I am too tired of spending time reviewing your absolutely invalid PRs, of your opened-closed-reopened issues, so I will have to ban you for spamming here and spreading the misinformation about the project outside.
